spec/requests/users_spec.rb

```diff
@@ -1,7 +1,45 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
+  before(:all) do
+    @user = User.new(name: 'Bianca')
+    @user.save
+  end
+
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get users_path
+    end
+
+    it 'response status correct' do
+      expect(response.status).to eq(200)
+      expect(response).to be_successful
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'response body includes correct placeholders text' do
+      expect(response.body).to include('List of Users')
+    end
+  end
+
+  describe 'GET /show' do
+    before do
+      get user_path(@user)
+    end
+
+    it 'renders status correct' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      expect(response).to render_template(:show)
+    end
```

Comment:
- [ ] Please 🙏🏽 correct the test for the correct TDD.

```ruby
Users GET /index renders the index template
     Failure/Error: expect(response).to render_template(:index)
     
     NoMethodError:
       assert_template has been extracted to a gem. To continue using it,
               add `gem 'rails-controller-testing'` to your Gemfile.
     # ./spec/requests/users_spec.rb:20:in `block (3 levels) in <main>'

  4) Users GET /show renders the show template
     Failure/Error: expect(response).to render_template(:show)
     
     NoMethodError:
       assert_template has been extracted to a gem. To continue using it,
               add `gem 'rails-controller-testing'` to your Gemfile.
     # ./spec/requests/users_spec.rb:38:in `block (3 levels) in <main>'
```

Reply:
@franclobo Can you help me with the problem? I can't fix the problem when I try to use render_template. 
Can you guide me to solve it? Thank you